go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/layer5io/meshery-istio
 go 1.13
 
 replace (
+	// github.com/docker/docker => github.com/moby/moby v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible
```

Comment:
Can we remove these?

Reply:
some machine require this, so I've put it for those who need it. Like Mac, arm64 etc.